### PR TITLE
TELCODOCS-1867 - remove configuration step

### PR DESCRIPTION
**Pull request by slovern**
[TELCODOCS-1867](https://issues.redhat.com//browse/TELCODOCS-1867) - remove configuration step

Version(s): 4.17

Issue: https://issues.redhat.com//browse/TELCODOCS-1867

Link to docs preview: [Configuring the integration of MetalLB and FRR-K8s](https://82163--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/metallb/metallb-frr-k8s.html)

QE review:
- [x] QE has approved this change.

Additional information: This is the removal of the section [Activating the integration of MetalLB and FRR-K8s](https://docs.openshift.com/container-platform/4.16/networking/metallb/metallb-frr-k8s.html#nw-metallb-configuring-frr-k8s_configure-metallb-frr-k8s)  as this is now the default backend and does not require activation.

### CODE REVIEW

File: networking/metallb/metallb-frr-k8s.adoc [hunk 1]
```diff
@@ -8,14 +8,12 @@ toc::[]
 
 FRRouting (FRR) is a free, open source internet routing protocol suite for Linux and UNIX platforms.
 `FRR-K8s` is a Kubernetes based DaemonSet that exposes a subset of the `FRR` API in a Kubernetes-compliant manner.
-As a cluster administrator, you can use the `FRRConfiguration` custom resource (CR) to configure `MetalLB` to use `FRR-K8s` as the backend.
+As a cluster administrator, you can use the `FRRConfiguration` custom resource (CR) to configure the way `MetalLB` uses `FRR-K8s`.
```
Comment (fedepaol) [inline networking/metallb/metallb-frr-k8s.adoc:11]: this is more "to leverage the FRR instance beyond the features offered by MetalLB"

Comment (slovern) [reply networking/metallb/metallb-frr-k8s.adoc]: As a cluster administrator, you can use the FRRConfiguration custom resource (CR) to avail of some of the FRR services not provided by MetalLB, for example, receiving routes.

**Comment by slovern** [comment]
thanks for the review @mburke5678 
I've taken on your suggestion

**Comment by slovern** [comment]
thanks for the merge @michaelryanpeter

**Pull request merged**